aws/microservices/cluster.yml

```diff
@@ -8,45 +10,69 @@ metadata:
   name: thingsboard
   # Specify the desired aws region. Don't forget to update the 'availabilityZones' parameter to match the region.
   region: us-east-1
-  version: "1.22"
+  version: "1.28"
 
+iam:
+  withOIDC: true
+  serviceAccounts:
+  - metadata:
+      name: aws-load-balancer-controller
+      namespace: kube-system
+    wellKnownPolicies:
+      awsLoadBalancerController: true
 
 managedNodeGroups:
   - name: tb-node
-    instanceType: m5.xlarge
+    instanceType: m6a.xlarge
     desiredCapacity: 3
     maxSize: 3
```

Comment:
it is better
maxSize = desiredCapacity + 1
Otherwise, you will not be able to upgrade or recover nodes smoothly